app/assets/stylesheets/to_change_everything/_portugues.scss

```diff
@@ -0,0 +1,511 @@
+.portugues {
+  @import "common/font_styling";
+  /*--- Highlight language in mobile language menu ---*/
+  /*--- TODO: make sure the div selection (e.g. #lang1) matches what was added in
+  app/views/layouts/to_change_everything.html.erb ---*/
+  #lang1  li  > a[href="/tce/portugues"] {
```

Comment:
change this to `#lang2` as portugues is in that div [here][0]

[0]:https://github.com/crimethinc/website/blob/master/app/views/layouts/to_change_everything.html.erb#L188-L197

Reply:
@astronaut-wannabe If you add both selectors to this in the generator, then this step can be ignored on future locales.

```
  #lang2  li  > a[href="/tce/portugues"],
  #lang1  li  > a[href="/tce/portugues"] {
```

Reply:
@veganstraightedge opened a PR here https://github.com/crimethinc/website/pull/975